apps/parsley/src/components/LogRow/CaretToggle/index.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
 }
 const CaretToggle: React.FC<Props> = ({ onClick, open }) => (
   <IconButton
-    aria-label="Click to open or close section"
+    aria-label={`Click to ${open ? "close" : "open"} section`}
```

Comment:
I believe aria-label elements don't need the interaction instructions
```suggestion
    aria-label={`${open ? "Close" : "Open"} section`}
```